spec/Cabin/Infotainment.vspec

```diff
@@ -267,4 +268,28 @@ SmartphoneProjection.SupportedMode:
   allowed: [ 'ANDROID_AUTO', 'APPLE_CARPLAY', 'MIRROR_LINK', 'OTHER' ]
   description: Supportable list for projection.
 
+SmartphoneScreenMirroring:
+  type: branch
+  description: All smartphone screen mirroring actions.
+  comment: Smartphone screen mirroring mirrors the whole screen of the Smartphone on the vehicle infotainment system.
+
+SmartphoneScreenMirroring.Active:
+  datatype: string
+  type: actuator
+  allowed: [ 'NONE', 'ACTIVE', 'INACTIVE' ]
+  description: Mirroring activation info.
+  comment: NONE indicates that mirroring is not supported.
+
+SmartphoneScreenMirroring.Source:
+  datatype: string
+  type: actuator
+  allowed: ['USB', 'BLUETOOTH', 'WIFI']
```

Comment:
Consider using the following instead:
```suggestion
  allowed: ['USB', 'BLUETOOTH', 'WI-FI']
```

Although many folks tend to write "Wifi" or "WiFi", the correct spelling is "Wi-Fi": https://www.wi-fi.org/

Reply:
I have no strong opinion but the literals used here correspond to what we have in https://github.com/COVESA/vehicle_signal_specification/blob/master/spec/Cabin/Infotainment.vspec#L258 

```
SmartphoneProjection.Source:
  datatype: string
  type: actuator
  allowed: ['USB', 'BLUETOOTH', 'WIFI']
  description: Connectivity source selected for projection.
```

So if we are to change to `WI-FI` we should also consider changing `Vehicle.Cabin.Infotainment.SmartphoneProjection.Source` (and then comes the question on how to handle backward compatibility, just adding `WI-FI` and keep `WIFI` for now, but state that `WIFI`may be removed later)

I suggest we discuss it at a meeting

Reply:
Thanks for your comment.

Personally, I don't have a strong opinion on this matter either. I also agree that following the global standards is important.

However, I'm little bit confused about whether it's appropriate to use "WI-FI" for data naming. When I see the android document, "WIFI" is used as a data type name, and "Wi-Fi" is used for descriptive text in the document. (https://source.android.com/docs/core/ota/modular-system/wifi)

Typically, hyphens ('-') are not used in data type names in VSS. Therefore, how about we keep using "WIFI" as it is?

Reply:
I think changing "WIFI" to "WI-FI" is not too difficult since we only need to update the naming in SmartphoneProjection. More important thing is making the right decision regarding the correct naming between WIFI and WI-FI.

Reply:
At the VSS meeting earlier this week we agreed that keeping `WIFI` is the best option for now, as we then do not need to change the other existing signal that use `WIFI`.